### PR TITLE
Rearranged about the soybase genetic maps under community

### PR DESCRIPTION
**Pull request by weihuang12**
Removed "about the SoyBase Genetic maps" under community, and added a link "Details about the SoyBase genetic maps" to MAPS. Changed GCViT URL link under MAPS.

### CODE REVIEW

**Review by maxglycine — CHANGES_REQUESTED** [review]
OK, under the menu item "about & contact", I found a link "genetic maps and genome nomenclature" link.  That link lead me to the page "About the SoyBase Genetic Maps" which talks about the genetic map and sequence map (GmGWAS).   This page should also be linked to from  "Maps" =>"Genetic Map Resources for Soybean". There is a dead link "Details about the SoyBase genetic maps that should go to the "About the SoyBase Genetic Maps" (/about/about_maps) page ( I think).  As far as I can see, there is no link to the page "Genetic Maps and Genome Nomenclature" from the "Community" menu item.

**Comment by weihuang12** [comment]
To Rex's request:  The "Details about the SoyBase genetic maps" under "MAPS" is linked to "Genetic maps and map nomenclature" under the menu item "about & contract",  and is not a dead link.
 I removed "About the SoyBase Genetic Maps" under "Community" ( I thought it was redundant), but I will add it back with link to "Genetic maps and map nomenclature".

**Review by StevenCannon-USDA — COMMENTED** [review]
The suggestion to move "About the SoyBase Genetic Maps" from the community page (under the "Communications" heading) was from me. @maxglycine - do you really want an "about maps" page under "Communications"? It seems to me that the maps page is a better place to link that from.

I also suggest removing the link to the "[2009 Soybean Genomic Sequence Assembly Announcement](https://soybase.org/community/about_sequence_map.php)" altogether, since that is now VERY old news.

**Comment by maxglycine** [comment]
@StevenCannon-USDA I did not suggest putting the link under “community” in the “communications” sub-section. I believe @weihuang12 did that. If it was up to me, the only reference would be under the “maps” menu item.

yes, we should drop the https://soybase.org/community/about_sequence_map.php link for just the reason you pointed out.

Anything else?

File: _data/community.yml
Comment (StevenCannon-USDA) [inline _data/community.yml:1]: On this file, please delete lines 31 and 32: /about/about_maps and "About the SoyBase Genetic Maps"
and also "2009 Soybean Genomic Sequence Assembly Announcement" and link.

**Review by StevenCannon-USDA — COMMENTED** [review]
Sorry for the confusion, Wei.
Two more changes to make on the community page: remove the about_maps link and the link to "2009 Soybean Genomic Sequence Assembly Announcement".

**Review by StevenCannon-USDA — APPROVED** [review]
Looks good to me. I'll approve and merge.

**Review by StevenCannon-USDA — APPROVED** [review]

**Review by StevenCannon-USDA — APPROVED** [review]

**Comment by StevenCannon-USDA** [comment]
Hmm. Seems that @maxglycine also has to review and approve. Rex - I think that is because you requested changes. My approval wasn't sufficient to allow a merge.

**Review by maxglycine — APPROVED** [review]

**Comment by maxglycine** [comment]
I have approved

**Comment by maxglycine** [comment]
@StevenCannon-USDA and @weihuang12 I have approved

**Comment by StevenCannon-USDA** [comment]
Thanks, @weihuang12 and @maxglycine .
I have merged this branch and will delete it now.

**Pull request merged**